src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,25 @@
-export const PostList = () => (
-  <>Put the list here</>
-);
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({
+  postsFromServer, commentsFromServer, usersFromServer,
+}) => {
+  function getUserById(userId) {
```

Comment:
Consider defining this function outside of the component. It's a helper function, so we should have the possibility to reuse it